erpnext_com/www/docs/user/manual/en/setting-up/stock-reconciliation.md

```diff
@@ -11,26 +11,35 @@ Stock Reconciliation is used for:
 * Posting opening stock
 * Reconciling book and actual stock
 
->Stock Reconciliation can be used for above listed purposes only for non-serialized items. For serialized and batched items Stock Entry can be used.
+###1. How to Create a Stock Reconciliation to Post Opening Stock
 
-**Difference between Serialized and Non-serialized Items.**
+####1.1. For Non-serialized Items
 
-A serial number is a unique, identifying number or group of numbers and
-letters assigned to an individual Item. Serialized items are generally high value items for which you need to manage warranty and service agreements. Mostly items like machinery, equipments and high-value electronics (computers, printers etc.) are serialized.
+Using stock reconciliation you can update the quantity of specific items in a warehouse as of specific time.
 
-Non Serialized items are generally fast moving and low value item, hence don't need tracking for each unit. Items like screw, cotton waste, other consumables, stationary products can be categorized as non-serialized.
```

Comment:
why delete? this info can be useful for new users...

Reply:
This was useful when we're using the stock reconciliation only for the Serialized items but now it will be use for both serialized and non serialized items.